src/schema/v2/filterArtworksConnection.ts

```diff
@@ -0,0 +1,134 @@
+import Artwork from "./artwork"
+import {
+  computeTotalPages,
+  createPageCursors,
+  pageToCursor,
+} from "./fields/pagination"
+import { pageable } from "relay-cursor-paging"
+import { convertConnectionArgsToGravityArgs, removeNulls } from "lib/helpers"
+import { connectionFromArraySlice, connectionDefinitions } from "graphql-relay"
+import { GraphQLFieldConfig } from "graphql"
+
+import { ResolverContext } from "types/graphql"
+import { includesFieldsOtherThanSelectionSet } from "lib/hasFieldSelection"
+import { FilterArtworksFields, filterArtworksArgs } from "./filter_artworks"
+
+// The connection fields here match the original implementation's,
+// except without `artworks_connection`.
+// TODO: Support proper thunk.
+const { artworks_connection, ...connectionFields } = FilterArtworksFields()
+
+// TODO: Update `graphql-relay` to support other GraphQL object
+// configurations, such as `interfaces`.
+const filterArtworksConnectionType = connectionDefinitions({
+  name: "FilterArtworks",
+  nodeType: Artwork.type,
+  // interfaces: [NodeInterface],
```

Comment:
This seems like possibly an update in the lib (it can accept all of the object configs, besides the specific connection ones, and pass those through).

Or, can maybe just be used via `filterArtworksConnectionType.interfaces = [NodeInterface]`. I didn't play with this really.

Reply:
Remind me, this is so we can refer to the connection and its filter state by a global ID?

You can’t mutate a graphql-js type, so the latter wouldn’t work.

Reply:
Yup! I'm wondering if we actually need this though, or if a node query would resolve properly anyway (will check it out).

If it's a blocker then I can patch-package the lib to allow more options to be passed in to the GraphQL object type created.